wagtailio/static/css/components/cookie-message.scss

```diff
@@ -0,0 +1,71 @@
+.cookie {
+    $root: &;
+    padding: 0px 10px 10px 10px;
+    position: fixed;
+    right: auto;
+    bottom: 0;
+    left: auto;
+    z-index: 13;
+    display: none; // Doesn't display unless there isn't a cookie
+    width: 100%;
+    pointer-events: none;
+
+    // Applied if cookie just added
+    &.inactive {
+        display: block;
+        transform: translate3d(0, 100%, 0);
```

Comment:
Very picky, but you could use `transform: translateY(100%);` instead as we aren't concerned with `X` or `Z` here.